tracekit.js

```diff
@@ -291,8 +302,8 @@ TraceKit.report = (function reportModuleWrapper() {
         // slow slow IE to see if onerror occurs or not before reporting
         // this exception; otherwise, we will end up with an incomplete
         // stack trace
-        setTimeout(function () {
-            if (lastException === ex) {
+        window.setTimeout(function () {
```

Comment:
Any specific reason to putting this to window? It might block users from using this in node?